### PR TITLE
Make strings const in `funcs.rs`

### PR DESCRIPTION
**Pull request by yuk1ty**
Replaced hard-coded strings with constants.

#266 

@mtshiba

### CODE REVIEW

File: crates/erg_compiler/context/initialize/funcs.rs [hunk 1]
```diff
@@ -21,75 +21,75 @@ impl Context {
...
-        let T = mono_q("T", instanceof(Type));
-        let U = mono_q("U", instanceof(Type));
-        let Path = mono_q_tp("Path", instanceof(Str));
-        let t_abs = nd_func(vec![kw("n", mono(NUM))], None, Nat);
+        let T = mono_q(TY_T, instanceof(Type));
+        let U = mono_q(TY_U, instanceof(Type));
+        let Path = mono_q_tp(PATH, instanceof(Str));
+        let t_abs = nd_func(vec![kw(KW_N, mono(NUM))], None, Nat);
         let t_all = func(
-            vec![kw("iterable", poly("Iterable", vec![ty_tp(Bool)]))],
```
Comment (GreasySlug) [inline crates/erg_compiler/context/initialize/funcs.rs:29]: The first letter seems to be capitalized("iterable" to "Iterable")
Wonder why the error didn't happen before
@mtshiba

Comment (yuk1ty) [reply crates/erg_compiler/context/initialize/funcs.rs:29]: Thanks, maybe I misplaced `KW_ITERABLE` to `Iterable` ones 🤔  I've fixed it and rerun CI.

Comment (GreasySlug) [reply crates/erg_compiler/context/initialize/funcs.rs:29]: Certainly test is passed, but please have mtshiba look at it again.

Comment (mtshiba) [reply crates/erg_compiler/context/initialize/funcs.rs:29]: I think nothing is wrong.
`iterable` is a parameter name, and `Iterable` is a type name for it.

**Comment by mtshiba** [comment]
Thanks!

**Pull request merged**